src/content/app/genome-browser/components/browser-track-config/BrowserTrackConfig.scss

```diff
@@ -48,3 +48,8 @@
     vertical-align: middle;
   }
 }
+
+.radio {
+  height: 15px;
+  width: 15px;
+}
```

Comment:
What's going on here? Why are these not set through the CSS variable, and why are they set at all? Do they need to be different from the default size?

Reply:
I missed these when checking where radio buttons were used. I'll look into this and see if it should be any different.

Reply:
We don't need this class so I will remove it

Reply:
✅